build/Versions.props

```diff
@@ -31,7 +31,7 @@
     <MicrosoftExtensionsDependencyModelVersion>2.0.0</MicrosoftExtensionsDependencyModelVersion>
     <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25908-02</NETStandardLibraryNETFrameworkVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
-    <NuGetBuildTasksPackVersion>4.5.0-preview2-4529</NuGetBuildTasksPackVersion>
+    <NuGetBuildTasksPackVersion>4.6.0-preview2-4745</NuGetBuildTasksPackVersion>
     <NuGetPackagingVersion>4.5.0-preview2-4529</NuGetPackagingVersion>
```

Comment:
@mishra14
Why is "NuGetPackagingVersion" at the older version? Is this an oversight in the updating process?

Reply:
@johnbeisner Why do you have 3 different versions for NuGet?

Reply:
@mishra14 
The question is for you to field: Why would your update-tool update one and not the others?

Reply:
@johnbeisner I do not understand the need for 3 different variables. We can fix our automated PR process to update all of those, but if all three have the same value, then why not just use 1?

Reply:
@mishra14 
"I do not understand the need for 3 different variables" - orchestrated build dictates the granularity of the packages

What is your process updating? And I'll base the others on the one your process is updating.